src/components/molecules/page-header/page-header.md

```diff
@@ -24,3 +24,33 @@
   }}
 />
 ```
+
```

Comment:
Let's add an **Example** title here, and then the following:

### Single actions
A Page Header can have a single primary action:

Reply:
Done!